docs/manual/build_system/recipe_fedora.qbk

```diff
@@ -0,0 +1,51 @@
+[/=============================================================================
+   Copyright (c) 2015 Adrian Serio
+
+   Distributed under the Boost Software License, Version 1.0. (See accompanying
+   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+==============================================================================/]
+
+[section:fedora_installation How to Install __hpx__ on Fedora Distributions]
+
+[note This section of the manual is based off of our collaborators Patrick
+Diehl's blog [@http://diehlpk.github.io/2015/08/04/hpx-fedora.html Installing
+HPX on Fedora 22].]
+
+[teletype]
+
+* Install all packages for minimal installation
+``
+   sudo dnf gcc-c++ install cmake boost-build boost boost-devel hwloc-devel \
```

Comment:
shouldnt this be sudo dnf install gcc-c++ cmake boost-build ...... instead of sudo dnf gcc-c++ install ..... ?